docs/package_firewall/cargo.md

```diff
@@ -0,0 +1,116 @@
+# Cargo Package Firewall
+
+The Cargo package firewall is based on Cargo's sparse [Registry Index API].
+
+[Registry Index API]: https://doc.rust-lang.org/cargo/reference/registry-index.html
+
+## Configuration
+
+All configuration options will require a Phylum API key, since Phylum's package
+firewall requires authentication. You can find out how to generate one in our
+[API Keys documentation].
+
+In the following examples, all API keys will be represented as
+`<PHYLUM_API_KEY>`, so make sure to replace them with your generated key.
+
+Additionally, if the default [policy] is not sufficient, a group can be passed
+to evaluate all packages against the group's policy. To do this, just replace
+`<PHYLUM_GROUP>` with the desired group name. The supplied API key **must** have
+access to this group.
+
+If Phylum's default [policy] is sufficient, you can remove all instances of
+`<PHYLUM_GROUP>`.
+
+[API Keys documentation]: ../knowledge_base/api-keys.md#generate-an-api-key
+[policy]: ../knowledge_base/policy.md
+
+### cargo
+
+To use Phylum's Cargo registry, it first needs to be added to a
+[`.cargo/config.toml` file]. On UNIX you can use `~/.cargo/config.toml` to make
+the registry available to all projects.
+
+```toml
+[registries]
+phylum = { index = "sparse+https://cargo.phylum.io/" }
+```
+
+This makes the Phylum registry available to all Cargo projects, however by
+default Cargo will still use the official registry. To ensure all requests go
+through Phylum, you'll have to override the crates.io registry in the
```

Comment:
Just a suggestion, but it might look and read more easily if "crates.io" is either turned into a link or put in backticks: `crates.io`. If the change is made here, then it should be made in the other spot it shows up a few lines down.